src/website/app/demos/Menu/bestPractices.js

```diff
@@ -0,0 +1,286 @@
+/**
+ * Copyright 2017 CA
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/* @flow */
+import React from 'react';
+import { createStyledComponent, pxToEm } from '../../../../utils';
+import Menu, { MenuDivider, MenuGroup, MenuItem } from '../../../../Menu';
+
+const DemoLayout = createStyledComponent('div', ({ theme }) => ({
+  backgroundColor: theme.color_gray_10,
+  padding: theme.spacing_double,
+
+  '& > div': {
+    backgroundColor: 'white',
+    width: pxToEm(224)
+  }
+}));
+
+export default {
+  menu: [
+    {
+      type: 'do',
+      title: 'use the appropriate variant for your intent',
+      example: (
+        <DemoLayout>
+          <Menu>
+            <MenuItem>Share</MenuItem>
+            <MenuItem>Edit</MenuItem>
+            <MenuItem>Duplicate</MenuItem>
+            <MenuItem variant="danger">Delete</MenuItem>
+          </Menu>
+        </DemoLayout>
+      ),
+      description: `Match Mineral UI's pre-defined variants with semantic purpose.`
+    },
+    {
+      type: 'do',
+      title: 'use clear labeling for option messaging',
+      example: (
+        <DemoLayout>
+          <Menu>
+            <MenuItem secondaryText="Cmd + L">Select Line</MenuItem>
```

Comment:
None of the demo examples use 'Title Casing', so these practices are inconsistent. I would imagine 'Sentence casing' would be more appropriate for MenuItems.

Reply:
updated